src/app/_components/EventLinkModal.js

```diff
@@ -47,6 +47,7 @@ export function EventLinkModal({ isOpen, onClose, link }) {
       className="w-64 opacity-100 lg:w-fit lg:bg-primary-200"
       isCloseButton={isDesktop}
       layout={isDesktop}
+      scrollY
```

Comment:
Do we have this prop?

Reply:
It was there, but eventually I came up with another idea) thanks for noticing it!